Aliucord/src/main/java/com/aliucord/settings/Plugins.java

```diff
@@ -234,6 +234,8 @@ public void onToggleClick(ViewHolder holder, boolean state, int position) {
             String name = data.get(position).getName();
             PluginManager.togglePlugin(name);
             holder.card.settingsButton.setEnabled(state);
+            var p = Objects.requireNonNull(PluginManager.plugins.get(name));
```

Comment:
data.get(position) = plugin, so just change the first line in this method instead of getting it like this